### PR TITLE
Fix grammatical issue with 'create workload object'

### PR DESCRIPTION
**Pull request by andrewreeman**
Using indefinite article for 'create a workload object'

### CODE REVIEW

**Comment by natalisucks** [comment]
Hi @andrewreeman – welcome to Kubernetes. This PR is classified as a [trivial edit](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits), so I'd like to suggest that you look through the rest of the source for further fixes and improvements, given we are a small reviewer crew where fixes like these give more work than value. Thanks in advance.

File: content/en/docs/concepts/workloads/controllers/_index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Ultimately, your applications run as containers inside
 Pods would be a lot of effort. For example, if a Pod fails, you probably want to
 run a new Pod to replace it. Kubernetes can do that for you.
 
-You use the Kubernetes API to create workload
+You use the Kubernetes API to create a workload
 {{< glossary_tooltip text="object" term_id="object" >}} that represent a higher level
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/_index.md:17]: and maybe this:
```suggestion
{{< glossary_tooltip text="object" term_id="object" >}} that represents a higher level
```
?

Comment (andrewreeman) [reply content/en/docs/concepts/workloads/controllers/_index.md]: Taken that suggestion on board and added a new commit. Also slightly reworded so it flows better (imo).

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

**Pull request merged**